schemas/system/Identifier.schema

```diff
@@ -0,0 +1,7 @@
+System({
+  Family: Enum('Registry', 'Dictionary', 'File', 'Cache'),
+  Status: Enum('Preallocated', 'Intialization', 'Actual', 'Historical'),
+  Lock:  Enum('Free', 'Locked'),
+  Create: { domain: 'Timestamp' },
```

Comment:
Maybe `Created` and `Changed`? As I assume they denote creation/last change time?

Reply:
No, in databases we use singular infinitive in entity and attribute naming. In *nix systems we have access, change, modify time, not accessed, changed, modified time.